lib/hw_id/Kconfig

```diff
@@ -27,7 +27,7 @@ config HW_ID_LIBRARY_SOURCE_UUID
 	  This option requires modem firmware v1.3.0 or higher.
 
 config HW_ID_LIBRARY_SOURCE_BLE_MAC
-	bool "BLE MAC address"
+	bool "Bluetooth MAC address"
```

Comment:
Please use preferably the "Bluetooth Device Address" term

Reply:
I'll split this change into a separate PR, it's not directly related (was just a drive-by fixup) 👍

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/25576